scripts/scr_inventory_draw.gml

```diff
@@ -11,13 +11,11 @@ if (active)
     var y2 = y1 + 9 * (heightBorder);
     y1 = y1 + heightBorder;
     
-    draw_set_color(c_black);
     draw_set_alpha(0.8);
+    draw_set_color(c_black);
     draw_rectangle(x1,y1,x2,y2,0);
-    draw_set_alpha(1);
-    
-    // Draw the items
     
+    // Draw the items    
```

Comment:
If you add constants for the row and column sizes, this comment won't be necessary anymore, you could add a `// TODO(isaac): add code to draw items here` instead.

Reply:
Main intent was so that I can just glance through the comments and know where I should be looking to add/edit the code. I guess I still need the todo there though.

Reply:
No triaging for now. I'm going to assume I'll be fixing most of the code TODO's anyway.